### PR TITLE
non-usethis dependent template renderer

### PR DESCRIPTION
**Pull request by thebioengineer**
Simplify templating to remove having to get around {here] tools in usethis::use_template

### CODE REVIEW

**Comment by thebioengineer** [comment]
@elimillera - if you wouldnt mind reviewing this and approving to merge prior to updating the "vt_use_*" functions to templates, I feel as though this will make it easier

File: R/vt_template.R [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+#'
+render_template <- function(template, output = template, data = list()){
+
+  template_path <- file.path(
+    system.file("templates", package = "valtools"),
+    template)
+
+  if(!file.exists(template_path)){
+    abort(paste0(c("Error during creation of template `",template,"`. Error: ",
+                   e, sep = "\n")),
```
Comment (elimillera) [inline R/vt_template.R:22]: Can this message be made more specific? I don't think `e` exists in this context.

File: R/vt_template.R [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    output_text <- whisker.render(
+      template = template_text,
+      data = data)
+
+    file_con <- file(output)
+
+    writeLines(
+      output_text,
+      con = file_con
```
Comment (elimillera) [inline R/vt_template.R:38]: I'm thinking we might want to have a `finally{close(file_con)}` for all of the `writeLines`. Thoughts?

**Review by elimillera — COMMENTED** [review]
Hey Ellis, A couple of questions. I'll make sure to use this for my PR.

File: R/vt_template.R (hunk 1 above)
Comment (thebioengineer) [reply R/vt_template.R:22]: :facepalm: yes, it should be a clearer "template does not exist". Of course e doesn't exist

File: R/vt_template.R (hunk 2 above)
Comment (thebioengineer) [reply R/vt_template.R:38]: Good point. Yes, add that in

**Comment by thebioengineer** [comment]
Thanks @elimillera . I'll make the edits post haste

File: R/vt_template.R (hunk 2 above)
Comment (elimillera) [reply R/vt_template.R]: Looks good. I also added this as issue #30. I'll submit a PR for that once we get everything else all merged in.

**Review by elimillera — APPROVED** [review]

**Pull request merged**